model/api/src/main/java/org/jboss/aerogear/unifiedpush/api/validation/DeviceTokenValidator.java

```diff
@@ -72,6 +78,8 @@ public boolean isValid(Installation installation, ConstraintValidatorContext con
                 return WINDOWS_DEVICE_TOKEN.matcher(deviceToken).matches();
             case SIMPLE_PUSH:
                 return SIMPLE_PUSH_DEVICE_TOKEN.matcher(deviceToken).matches();
+            case ADM:
+                return ADM_DEVICE_TOKEN.matcher(deviceToken).matches();
```

Comment:
Can we have a test case for this ? Something like:
https://github.com/aerogear/aerogear-unifiedpush-server/blob/master/model/jpa/src/test/java/org/jboss/aerogear/unifiedpush/jpa/InstallationDaoTest.java#L435